python/ray/includes/gcs_client.pxi

```diff
@@ -589,6 +575,23 @@ cdef void assign_and_decrement_fut(result, void* fut_ptr) with gil:
         else:
             fut.set_result(ret)
     finally:
+        # Refcounting in Cython is really pesky, because Cython makes a bunch of + and -
```

Comment:
Also add a link to the cython doc where the info is from

Reply:
added in cpp